apps/web/app/admin/(items)/_components/AddCategoryForm.tsx

```diff
@@ -0,0 +1,45 @@
+"use client";
+
+import { useMutation, useQueryClient } from "@tanstack/react-query";
+import { Button, Input, notify } from "@ui/index";
+import { useForm } from "react-hook-form";
+import { postNewCategory } from "@/api/meetings";
+
+export default function AddCategoryForm(): JSX.Element {
+  const { register, handleSubmit } = useForm();
+  const queryClient = useQueryClient();
+
+  const mutation = useMutation({
+    mutationFn: async (payload: Record<string, string>) => {
+      return await postNewCategory(payload);
+    },
+    onSuccess: async () => {
+      notify({ type: "success", message: "카테고리가 추가되었습니다!" });
+      await queryClient.invalidateQueries({ queryKey: ["categories"] });
+    },
```

Comment:
```suggestion
    onSuccess: async () => {
      notify({ type: "success", message: "카테고리가 추가되었습니다!" });
      void queryClient.invalidateQueries({ queryKey: ["categories"] });
    },
```
여기도 비동기 처리가 필요할까 싶기도 하고..
queryKey는 파일에 상수화해서 관리해보는건 어때?

Reply:
web/lib/queryKey.ts에 만들어놓았으 여기에 적으면 될듯 해!